extensions/wikia/Chat2/ChatWidget.class.php

```diff
@@ -192,23 +189,19 @@ function () use ( $userName ) {
 				// get stats for edit count and member since
 				$user = User::newFromName( $userName );
 
-				if ( $user instanceof User ) {
+				if ( $user instanceof User && $user->isLoggedIn() ) {
 					$userStatsService = new UserStatsService( $user->getId() );
```

Comment:
I don't like the name `isLoggedIn()` as it suggests `isCurrentlyLoggedIn()` while it actually is `! isAnon()`. How about using `! isAnon()` which suggests exactly what it says?